lib/warder/coffee_lint_runner/message_line_formatter.rb

```diff
@@ -0,0 +1,31 @@
+module Warder
+  # responsible for run coffeescript style guide validation
+  class CoffeeLintRunner
+    # responsible for human readable error message representation
+    class MessageLineFormatter
+      MESSAGE_KEY = 'message'.freeze
+      CONTEXT_KEY = 'context'.freeze
+      LEVEL_KEY = 'level'.freeze
+      LINE_NUMBER_KEY = 'lineNumber'.freeze
+
+      def initialize(result)
+        @result = result
+      end
+
+      def format
+        msg = [@result[MESSAGE_KEY], @result[CONTEXT_KEY]].join('. ')
+        [level, line_number, msg].join(': ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.